leo/cli/commands/run.rs

```diff
@@ -66,7 +66,9 @@ fn handle_run<N: Network>(command: LeoRun, context: Context) -> Result<<LeoRun a
     let mut inputs = command.inputs;
 
     // Compose the `run` command.
-    let mut arguments = vec![SNARKVM_COMMAND.to_string(), command.name];
+    // The argument "--" is the separator, used to make sure snarkvm doesn't try to parse negative
```

Comment:
Does this need to be done for `LeoExecute` as well?
Are the existing tests sufficient?